codecov.yml

```diff
@@ -12,7 +12,7 @@ coverage:
     patch:
       default:
         # minimum coverage ratio that the commit must meet to be considered a success
-        target: 70%
+        target: 0%
```

Comment:
Should we remove the patch coverage target?
If we want to exclude some method from testing there are comments we can use so the coverage tool does not count those ([reference](https://github.com/gotwarlost/istanbul/blob/master/ignoring-code-for-coverage.md))

Reply:
We are still working on the design for the tests implementation on the wallet-desktop, we will eventually need the patch coverage target, so I think it's OK to leave it at 0 for now (since we really don't have any tests)